lib/rules/utils/ast-utils.js

```diff
@@ -853,14 +853,7 @@ module.exports = {
     isDirectiveComment(node) {
         const comment = node.value.trim();
 
-        return (
-            node.type === "Line" && comment.indexOf("eslint-") === 0 ||
-            node.type === "Block" && (
-                comment.indexOf("global ") === 0 ||
-                comment.indexOf("eslint ") === 0 ||
-                comment.indexOf("eslint-") === 0
-            )
-        );
+        return /^(eslint[- ]|(globals?|exported) )/u.test(comment);
```

Comment:
```suggestion
        return /^(?:eslint[- ]|(?:globals?|exported) )/u.test(comment);
```

as we don't need to capture anything